jobbergate-cli/jobbergate_cli/subapps/job_scripts/app.py

```diff
@@ -289,16 +289,8 @@ def render(
         actual_value=submit,
     )
 
-    download = question_helper(
-        question_func=typer.confirm,
-        text="Would you like to download the job script files?",
-        default=True,
-        fast=fast,
-        actual_value=download,
-    )
-
-    if download:
-        download_job_script_files(job_script_result.job_script_id, jg_ctx)
+    if download and settings.SBATCH_PATH is None:  # on-site submission will download the job script files anyway
```

Comment:
This function will download the files if required by the user in remote mode, `create_job_submission` will download them anyway in on-site mode, so there is no need to pass it to the function. This is not ideal, but we need it at the `create-job-script` level since the submission is optional here.